analyses/independent-samples/independent_rna_samples.R

```diff
@@ -30,23 +31,21 @@
 independent_rna_samples <- function(independent_dna_sample_df, 
                                 histology_df,
                                 match_type = c("independent_dna", "independent_dna_plus_only_rna"),
-                                tumor_description_rna_only = c("primary","primary_plus"),
+                                tumor_description_rna_only = c("primary", "primary_plus"),
                                 seed){
   match_type <- match.arg(match_type)
   tumor_description_rna_only <- match.arg(tumor_description_rna_only)
   if(!missing(seed)){set.seed(seed)}
-  primary_descs <- c("Initial CNS Tumor", "Diagnosis")
-  
-  
+  primary_descs <- c("Initial CNS Tumor", "Diagnosis", "Primary tumor")
```

Comment:
```suggestion
  primary_descs <- c("Initial CNS Tumor", "Primary tumor")
```

Reply:
Thanks @jharenza! Changes are made accordingly - please check whether this is what you want. I haven't deleted the polya and stranded RNA independent output just for references. We can delete them later.

Reply:
Could you please remove the polya and stranded RNA files, since we don't want to keep extra files in `results` that cannot be created with the current code in the analysis folder.

Reply:
Sure. Done